### PR TITLE
Add a simple interactive UI

### PR DESCRIPTION
**Pull request by couchand**
This commit adds a stupidly simple interactive UI for comparing test
runs.  Use the same `roachperf web` command, and simply provide more
than two directories to enter interactive mode.

### CODE REVIEW

**Comment by tschottdorf** [comment]
Give me a screenshot or give me 💀

**Comment by couchand** [comment]
screen shot 2017-10-04 at 5 12 13 pm

**Comment by couchand** [comment]
Is there any reason the fields on `testRun` aren't exported?  I can avoid duplicating them here if I change them to be exported over there.

**Comment by petermattis** [comment]
> Is there any reason the fields on testRun aren't exported? I can avoid duplicating them here if I change them to be exported over there.

No reason. I default to unexported until I need to export. There is need now, so let's export them.

File: web.go [hunk 1]
```diff
@@ -152,3 +161,212 @@ const webHTML = `<html>
...
+      {{- range $j, $d := . }}{{ if $j }},{{ end }}
+        {
+          "metadata": {
+            "bin": {{ .metadata.Bin }},
+            "cluster": {{ .metadata.Cluster }},
+            "nodes": {{ .metadata.Nodes }},
+            "env": {{ .metadata.Env }},
+            "args": {{ .metadata.Args }},
+            "test": {{ .metadata.Test }},
+            "date": {{ .metadata.Date }}
```
Comment (petermattis) [inline web.go]: FYI, the `Date` field was only added recently. Not sure how useful it will be.

Comment (couchand) [reply web.go]: I added it in to have something to distinguish tests with the same binary.  That being said, presumably it's not interesting to compare tests with the same binary, so...

**Comment by couchand** [comment]
Yeah, it should be pretty quick to add a trend chart, I'll do that now.

**Comment by couchand** [comment]
screen shot 2017-10-05 at 11 13 01 am

**Pull request merged**